openflow13/match.go

```diff
@@ -239,11 +248,12 @@ func DecodeMatchField(class uint16, field uint8, data []byte) util.Message {
 
 		if val == nil {
 			log.Printf("Bad pkt class: %v field: %v data: %v", class, field, data)
-			return nil
+			//Add error msg info for caller
```

Comment:
This comment is not necessary, returning formatted error messages is extremely common in Go